.github/workflows/release-tagging.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Tagging
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  tagging:
+    runs-on: ubuntu-latest
+    env:
+      EXCLUDED_PACKAGE_OA_RENDERERS: '!./packages/vckit-oa-renderers/'
+      EXCLUDED_PACKAGE_CREDENTIAL_OA: '!./packages/credential-oa/'
+      EXCLUDED_PACKAGE_EXPLORER: '!./packages/demo-explorer/'
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

Comment:
The `actions/checkout@v3` action uses `node16` under the hood, which is deprecated by GitHub. We should switch to `actions/checkout@v4`.